models/Movie.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const movieSchema = new Schema({
+  title: String,
+  genre: String,
+  plot: String,
+  //Is that what is meant with array of Object Ids?
+  //Is that the right ref? I call the Schema Celebrity to be included in the movie Schema
```

Comment:
Yes this is how you define the cast as an array of `Celebrity` Ids. Each element in this array is of type "ObjectId" and is simply an ID of a celebrity. You can even check in Compass that the Ids in the cast array match those of the celebrities.